page-objects/pages/DashboardPage.js

```diff
@@ -14,7 +24,58 @@ async function isTitleOfDashboard() {
 	await waitHelpers.waitUntilPageTitleContains(dashboardBtnOnNavigationLeftPanel);
 }
 
+async function isPrivateTasksAndDraftsListVisible(listName) {
```

Comment:
Maybe the name of the parameter could be more descriptive? listName is very general.